runtime/bindings/python/tests/test_inference_engine/test_core.py

```diff
@@ -119,15 +105,15 @@ def test_read_model_from_onnx_as_path():
     assert isinstance(func, Function)
 
 
-@pytest.mark.xfail("68212")
-def test_read_net_from_buffer():
-    core = Core()
-    with open(test_net_bin, "rb") as f:
-        bin = f.read()
-    with open(model_path()[0], "rb") as f:
-        xml = f.read()
-    func = core.read_model(model=xml, weights=bin)
-    assert isinstance(func, IENetwork)
+# @pytest.mark.xfail("68212")
```

Comment:
Why have you commented this test?

Reply:
IENetwork is not existing now. Test will be enabled again and re-written with ticket 68212.

Reply:
PR: https://github.com/openvinotoolkit/openvino/pull/8593